arches_lingo/src/arches_lingo/components/basic-search/SearchResult.vue

```diff
@@ -0,0 +1,67 @@
+<script setup lang="ts">
+import type { Concept } from "@/arches_lingo/types.ts";
+import { getItemLabel } from "@/arches_lingo/utils.ts";
+
+defineProps({
+    searchResult: {
+        type: Object,
+        required: true,
+    },
+});
+
+const getParentLabels = (item: Concept, preferredLanguage: string): string => {
+    const arrowIcon = " → ";
+
+    return item.parents.reduce((acc, parent, index) => {
+        const label = getItemLabel(parent as Concept, preferredLanguage);
+        if (label) {
+            return acc + (index > 0 ? arrowIcon : "") + label;
+        }
+        return acc;
+    }, "");
+};
+</script>
+
+<template>
+    <div
```

Comment:
Can we make this tabbable?

Reply:
Maybe? I know search results are accessible via the arrow keys by default. Primevue has been titchy with `tabindex` for me so far

Reply:
Ah, I forgot to check down arrow. Then this is totally fine.

> Primevue has been titchy with tabindex for me so far

yeah, they [didn't like it](https://github.com/primefaces/primevue/issues/5427) when I showed them what i was doing with <Tree>

Reply:
oh lol yeah that's not great 😂